src/Kmd.Logic.Gateway.Automation/ValidatePublishing.cs

```diff
@@ -30,7 +31,8 @@ public ValidatePublishing(HttpClient httpClient, LogicTokenProviderFactory token
         public async Task<ValidationResult> ValidateAsync(string folderPath)
         {
             var publishYml = File.ReadAllText(Path.Combine(folderPath, "publish.yml"));
-            var publishFileModel = new Deserializer().Deserialize<PublishFileModel>(publishYml);
+
```

Comment:
Remove the extra line

Reply:
Okay i will remove the extra line

Reply:
I did changes removed the line as specified.